app/controllers/edit_sessions_controller.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
 class EditSessionsController < ApplicationController
-  include Wicked::Wizard
```

Comment:
I am suspicious of this 🤔 I did read the commit, but I remember something funny happening if one is omitted

If tests pass though maybe it's fine

Reply:
I was suspicious as well, but [the README was pretty explicit about replacing the module](https://github.com/zombocom/wicked/blob/main/README.md#internationalization-of-urls-i18n), and I saw nothing untoward during both automated and exploratory testing.